app/models/node.rb

```diff
@@ -359,20 +359,6 @@ def scraped_image
     match&.split('src="')&.last&.split('"')&.first
   end
 
-  # was unable to set up this relationship properly with ActiveRecord associations
-  def drupal_content_field_image_gallery
-    DrupalContentFieldImageGallery.where(nid: nid)
```

Comment:
Ah its because that was also not a built-in method - we deleted it too! OK, will adapt to this too.

Reply:
OK, here it is:

https://github.com/publiclab/plots2/blob/4efd03579f3e87a47bc7c0a2cd3836d25d6b268c/db/migrate/20220828142008_remove_drupal_gallery_from_node.rb#L6-L11